### ISSUE TITLE
Code release for language models

### ISSUE THREAD

**Issue by shivmgg** [NM-sparsity/NM-sparsity#18]
Hi, 

Thanks for the awesome work. Can you please release the code for the language models?

Regards,
Shivam

**Comment by aojunzz** [comment]
Yes, our work can be applied to LLMs, such as LLaMA. However, we haven't refined the code. For related work on post-training pruning, you might want to refer to https://github.com/locuslab/wanda.
